### DISCUSSION TITLE
Missing Land Cover spatial object reported when testing GetFeature call for LandCoverUnit feature type

### DISCUSSION THREAD

**Discussion by robblis** [INSPIRE-MIF/helpdesk-validator#795] [Q&A] [answered]
Hi Validator Team,

We are testing the LandCoverVector test using the LandCoverUnit feature type.

We get the following error in the, Conformance class: "GML application schemas, Land Cover > Basic test >lc-gml.a.1: Land Cover feature in dataset" test 

_The XML documents representing the spatial data set do not contain a feature of any of the spatial object types in the 'Land Cover' application schemas. Therefore, the spatial data set cannot conform to this conformance class. If you have expected to find spatial objects from the application schema in the data set, please consult the statistics information to see the spatial object types that have been found._

Here is the service url: https://agsinspire1.esri.com/arcgis/rest/services/lcc/MapServer/exts/InspireFeatureDownload/service?REQUEST=GetFeature&SERVICE=WFS&VERSION=2.0.0&TYPENAMES=lcv:LandCoverUnit&COUNT=10&namespaces=xmlns(lcv,http://inspire.ec.europa.eu/schemas/lcv/4.0)

My assessment is this is a valid error for the LandCoverUnit feature type because it would not contain any sub-feature elements of 'Land Cover' spatial objects. Is there a way to ignore this test if the LandCoverUnit feature type is used or do we not test with LandCoverUnit feature type?

Thanks for any help you can give,
Robert

**Comment by fabiovinci** [comment, marked as answer]
Dear @robblis,

the scope of the basic test is to advise the user that the "main" feature type is not present in the dataset, as for the case of the  LandCoverDataset feature type.

This test may have a limitation when using a GetFeature request, as the response may contain only one specific feature type.

In your case, I would suggest to use 'only' the GetFeature request to the LandCoverDataset feature type since the related members contain the LandCoverUnit feature type.

https://agsinspire1.esri.com/arcgis/rest/services/lcc/MapServer/exts/InspireFeatureDownload/service?REQUEST=GetFeature&SERVICE=WFS&VERSION=2.0.0&TYPENAMES=lcv:LandCoverDataset&COUNT=10&namespaces=xmlns(lcv,http://inspire.ec.europa.eu/schemas/lcv/4.0)

**Comment by robblis** [comment]
Hi @fabiovinci ,

We will ignore the test using the GetFeature request with the LandCoverUnit.

Thanks,
Robert

**Comment by fabiovinci** [comment]
Hi @robblis,

I will close the issue and convert it into a discussion as it could be useful for other users.
